### PR TITLE
Update the README to fit with the new release

### PR DESCRIPTION
**Pull request by KerbalKore**
@CobaltWolf is the "nice" thing on the Big G bit ok with you?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
...
 - Spacelab (from Eyes) station parts
+- Parts for the Apollo-Venus mission
 - Vanguard rocket and probe
 - Scout rocket
 - Redstone rocket, with Sergeant clusters and Explorer 1
 - Juno II, IV rockets
-- Diamant rocket (All variants)
 - Thor-Delta rocket (Almost all variants can be made)
-- Delta 2 rocket (Including upper stage and a variety GEM SRBs)
+- Delta 2 and 3 rockets (Including upper stage and a variety GEM SRBs)
```
Comment (DeltaDizzy) [inline README.md]: *variety of GEM SRBs

Comment (DeltaDizzy) [inline README.md]: *variety of GEM SRBs

File: README.md [hunk 2]
```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
...
+- 4 meter Delta Cryogenic Second Stage (5 meter
```
Comment (DeltaDizzy) [inline README.md]: 5 meter what?

File: README.md [hunk 3]
```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
...
 - Atlas rocket (everything from Atlas A up through Atlas V 551)
 - Agena upper stage (All variants)
 - Vega (old Kerolox upper stage for Atlas)
 - Centaur D, G, G', T, V upper stages
 - Titan 1, 2, 3, 4 rockets, with SRMs and Transtage (No IUS yet!)
+- Config switching for LR87's, with alternate kerolox versions and plumes to match!
```
Comment (DeltaDizzy) [inline README.md]: Now this may be getting into grammar nazi territory but why does `LR87's` have a possessive apostrophe?

File: README.md [hunk 4]
```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
...
+- Peacekeeper/Minotaur SRB based launcher
+- Athena SRB based launcher
 - Saturn 1, 1B, V
 - Saturn 1C and Saturn Multibody from Eyes
 - New probe cores
 - New antennas and dishes
 - New science experiments 
-In total, nearly 400 parts!
+- High quality realplumes built with Plume Party FX by Zorg!
+- Full engine ignitor configs by RocketMan
```
Comment (DeltaDizzy) [inline README.md]: Their forum username is _Rock3tman

**Comment by KerbalKore** [comment]
@DeltaDizzy , Fixed the issues you pointed out.

File: README.md [hunk 5]
```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
...
-- Diamant rocket (All variants)
 - Thor-Delta rocket (Almost all variants can be made)
-- Delta 2 rocket (Including upper stage and a variety GEM SRBs)
+- Delta 2 and 3 rockets (Including upper stage and a variety of GEM SRBs)
+- 4 meter Delta Cryogenic Second Stage, modeled by Blowfish. (5 meter available in ReDIRECT)
 - Atlas rocket (everything from Atlas A up through Atlas V 551)
 - Agena upper stage (All variants)
 - Vega (old Kerolox upper stage for Atlas)
 - Centaur D, G, G', T, V upper stages
 - Titan 1, 2, 3, 4 rockets, with SRMs and Transtage (No IUS yet!)
```
Comment (DeltaDizzy) [inline README.md]: We have both IUS and the smaller TOS now.

File: README.md [hunk 6]
```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
...
+- Config switching for the LR87s, with alternate kerolox versions and plumes to match!
+- Config switching for the H1, the RS27, and many more! (All config switching provided courtesy of B9PS)
```
Comment (DeltaDizzy) [inline README.md]: Just to be on the safe side, why not make it B9PartSwitch?

File: README.md [hunk 7]
```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
...
+- Gemini Ferry
+- Big Gemini 69' (nice)
 - Apollo Block I and II spacecraft, as featured in REAL LIFE!
 - Apollo Block III, III+, IV, and V spacecraft, from Eyes Turned Skywards
 - AARDV "Aardvark" autonomous resupply vehicle, also from Eyes
-- Modular 1.875m station parts for Gemini-era space stations
+- Modular 1.875m and 1.5m station parts for Gemini-era space stations
 - Skylab station parts
 - Spacelab (from Eyes) station parts
+- Parts for the Apollo-Venus mission
```
Comment (DeltaDizzy) [inline README.md]: That kind of sounds off, and the technical name was "Manned Venus Flyby", though you can also note it is part of the AAP.

File: README.md (hunk 5 above)
Comment (KerbalKore) [reply README.md]: Already mentioned lower down; but i do need to remove that, thanks!

**Comment by KerbalKore** [comment]
all done @DeltaDizzy

File: README.md [hunk 8]
```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
...
 - Saturn 1C and Saturn Multibody from Eyes
 - New probe cores
 - New antennas and dishes
 - New science experiments 
-In total, nearly 400 parts!
+- High quality realplumes built with Plume Party FX by Zorg!
+- Full engine ignitor configs by RocketMan
+In total, nearly 500 parts and many features for those parts!
 
 Planned content includes massive Saturn rockets including unflown variants, launchers based on Atlas and Titan that never left the drawing board, and parts based on proposed projects for Gemini and Apollo.
```
Comment (zorg2044) [inline README.md]: I believe Atlas and Titan here are referring to SLV3X, SLV3K, CELV, and LDC, all since added.

**Pull request merged**